sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
       ],
     },
   ],
+
+  // Japanese sidebar nav
+  docsJapanese: [
+    {
+      type: 'doc',
+      label: 'ScalarDB ドキュメンテーションホーム',
+      id: 'index',
+    },
+    {
+      type: 'category',
+      label: 'ScalarDB について',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'overview',
+          label: '概要',
+        },
+        {
+          type: 'doc',
+          id: 'design',
+          label: 'デザイン',
+        },
+        {
+          type: 'doc',
+          id: 'requirements',
+          label: '要件',
+        },
+        {
+          type: 'doc',
+          id: 'database-configurations',
+          label: '基盤となるデータベースの設定',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-notes',
+          label: 'リリースノート',
+        },
+        {
+          type: 'doc',
+          id: 'releases/release-support-policy',
+          label: 'リリースサポートポリシー',
+        },
+        {
+          type: 'doc',
+          id: 'roadmap',
+          label: 'ロードマップ',
+        },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'クイックスタート',
+      collapsible: true,
+      items: [
+        {
+          type: 'doc',
+          id: 'quick-start-overview',
+          label: '概要',
+        },
+        {
+          type: 'category',
+          label: 'ScalarDB Core ライブラリを使用してトランザクションを実行してみる',
+          collapsible: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb',
+              label: 'ScalarDB Core を使用する',
+            },
+            {
+              type: 'doc',
+              id: 'getting-started-with-scalardb-by-using-kotlin',
+              label: 'Kotlin を使用する',
+            },
```

Comment:
I feel uncomfortable here because `ScalarDB Core` is our library name, while `Kotlin` is a programming language. How about `JavaでScalarDB Coreを使用する` and `KotilnでScalarDB Coreを使用する` ？

Reply:
Understood, thanks! I'll change this in a different PR since we need to update the English version as well.

Reply:
Per [@/feeblefakie's similar comment](https://github.com/scalar-labs/docs-scalardb/pull/762#discussion_r1889476229) as well, I ended up changing this in the following commit: 2e1a759abbf24cf51dac2946909fdf0a04b117c1